packages/lib-classifier/src/plugins/drawingTools/experimental/models/tools/FreehandLineTool/FreehandLineTool.js

```diff
@@ -7,22 +7,27 @@ const FreehandLineTool = types
     marks: types.map(FreehandLine),
     type: types.literal('freehandLine')
   })
-  .actions((self) => ({
-    createMark(mark) {
-      const newMark = FreehandLine.create(
-        Object.assign({}, mark, { toolType: self.type })
-      )
-      self.marks.put(newMark)
-      return newMark
-    },
+  .actions((self) => {
+    return {
+      createMark(mark, coors) {
+        const newMark = FreehandLine.create(
+          Object.assign({}, mark, { toolType: self.type })
+        )
+        newMark.initialize(coors);
+        self.marks.put(newMark)
+		
+        return newMark
+      },
 
-    handlePointerMove(event, mark) {
-      mark.initialDrag(event)
-    },
+      handlePointerMove(event, mark) {
+        //mark.initialDrag(event)
+      },
```

Comment:
whitespace.

Also, looks like a store action has been commented out for debugging.